Client/Clique/Common/Constants/Constants.swift

```diff
@@ -12,7 +12,24 @@ enum Constants {
     enum Service {
         private static let baseURL = "https://cliquewebservice20221115180920.azurewebsites.net/api/"
         static let registerURL = baseURL.appending("Authentication/RegisterUser")
-        static let loginURL = baseURL.appending("Authentication/LoginUser")    
+        static let loginURL = baseURL.appending("Authentication/LoginUser")
+        static let categoriesURL = baseURL.appending("Event/GetCategories")
+        static let currenciesURL = baseURL.appending("Event/GetCurrencies")
+        static let createEventURL = baseURL.appending("Event/CreateNewEvent")
+    }
+    
+    enum Alerts {
+        static let successfullyUpdatedMsg = "Succesfully updated."
```

Comment:
I would suggest changing this to `successfullyUpdatedMessage` :)

```suggestion
        static let successfullyUpdatedMessage = "Succesfully updated."
```